app/models/recipe.rb

```diff
@@ -6,4 +6,9 @@ class Recipe < ApplicationRecord
   validates :preparation_time, presence: true
   validates :cooking_time, presence: true
   validates :description, presence: true
+  validates :public, presence: true
+
+  def total_price
+    recipe_food.sum { |item| item.quantity * item.food.price }
+  end
```

Comment:
- [ ] Please look into this method. It is not properly computed. Please find a screenshot of the error below:'
![Screenshot 2023-02-16 at 3 40 22 PM](https://user-images.githubusercontent.com/48407831/219396368-75d6310e-294d-4af1-a9a7-a3bbd44dbdca.png)